src/js/components/Spinner/Spinner.js

```diff
@@ -68,7 +75,21 @@ const Spinner = forwardRef(
         }))
       : borderThemeProp;
 
-    // children will take precedence over theme attributes
+    // check the size of the pad plus border
+    // than multiple by 2 in order to compare to it to theme sizes
+    let spinnerPadBorder;
+    if (!rest.border) {
+      spinnerPadBorder =
+        2 *
+        (parseMetricToNum(
+          theme?.global?.edgeSize && theme.global.edgeSize[spinnerPad],
+        ) +
+          parseMetricToNum(
+            theme?.global?.edgeSize && theme.global.borderSize[normalizedSize],
```

Comment:
I'm not follow why we say if `!rest.border` --> shouldn't we be always be adding the border width, whether it's coming from a prop or the theme?

Reply:
think I need a separate check if `rest.border` then map through and get that  `size`

Reply:
so I added a check if someone has border size set in their  theme similar to how hpe theme works but if  someone passes  in `border` as a prop we should just let the `pad` be